### DISCUSSION TITLE
登录错误

### DISCUSSION THREAD

**Discussion by kk00529** [mikumifa/biliTickerBuy#434] [提问专区]
登录管理那里的配置文件导入后点击登录显示错误，同样的cpp抢票程序也是这个问题。
<img width="1079" alt="e9524c8fb32306e74090e7e8241723d" src="https://github.com/user-attachments/assets/1d8ce140-59f5-499d-a365-2eb5cdf39dd7">

**Comment by mikumifa** [comment]
控制台有报错吗

**Reply by kk00529** [reply]
'gbk' codec can't decode byte 0xad in position 20: illegal multibyte sequence
